Packs/CarbonBlackDefense/Integrations/CarbonBlackDefenseV2/CarbonBlackDefenseV2.py

```diff
@@ -0,0 +1,1169 @@
+    policy_api_secret_key = params.get('policy_api_secret_key')
+    organization_key = params.get('organization_key')
+
+    # if your Client class inherits from BaseClient, SSL verification is
+    # handled out of the box by it, just pass ``verify_certificate`` to
+    # the Client constructor
+    verify_certificate = not params.get('insecure', False)
+
+    # if your Client class inherits from BaseClient, system proxy is handled
+    # out of the box by it, just pass ``proxy`` to the Client constructor
+    proxy = params.get('proxy', False)
+
+    demisto.debug(f'Command being called is {command}')
+
+    try:
+        client = Client(
+            base_url=base_url,
+            verify=verify_certificate,
+            proxies=proxy,
+            api_key=api_key,
+            api_secret_key=api_secret_key,
+            policy_api_key=policy_api_key,
+            policy_api_secret_key=policy_api_secret_key,
+            organization_key=organization_key
+        )
+
+        commands = {
+            'cbd-create-policy': create_policy_command,
+            'cbd-get-policies': get_policies_command,
+            'cbd-get-policy': get_policy_command,
+            'cbd-set-policy': set_policy_command,
+            'cbd-update-policy': update_policy_command,
+            'cbd-delete-policy': delete_policy_command,
+            'cbd-add-rule-to-policy': add_rule_to_policy_command,
+            'cbd-update-rule-in-policy': update_rule_in_policy_command,
+            'cbd-delete-rule-from-policy': delete_rule_from_policy_command,
+            'cbd-find-events': find_events_command,
+            'cbd-find-events-results': find_events_results_command,
+            'cbd-find-events-details': find_events_details_command,
+            'cbd-find-events-details-results': find_events_details_results_command,
+            'cbd-find-processes': find_processes_command,
+            'cbd-find-processes-results': find_processes_results_command,
+            'cbd-alerts-search': alerts_search_command,
+            'cbd-get-alert-details': get_alert_details_command,
+            'cbd-device-search': device_search_command,
+            'cbd-device-quarantine': device_quarantine_command,
+            'cbd-device-unquarantine': device_unquarantine_command,
+            'cbd-device-background-scan': device_background_scan_command,
+            'cbd-device-background-scan-stop': device_background_scan_stop_command,
+            'cbd-device-bypass': device_bypass_command,
+            'cbd-device-unbypass': device_unbypass_command,
+            'cbd-device-policy-update': device_policy_update_command,
+            'cbd-device-update-sensor-version': device_update_sensor_version_command
+        }
+
+        if command == 'test-module':
+            return_results(test_module(client, params))
+        elif command == 'fetch-incidents':
+            fetch_time = demisto.params().get('first_fetch', '7 days')
+            fetch_limit = demisto.params().get('max_fetch', 50)
```

Comment:
```suggestion
            fetch_time = params.get('first_fetch', '7 days')
            fetch_limit = params.get('max_fetch', 50)
```

Please make sure you don't need no cast to int